Source/Menu/MainForm.cs

```diff
@@ -1158,28 +1159,26 @@ void ShowDetails()
             if (radioButtonModeTimetable.Checked)
             {
                 if (SelectedTimetableSet != null)
-                {
                     ShowDetail(catalog.GetStringFmt("Timetable set: {0}", SelectedTimetableSet), new string[0]);
-                }
+                // Description not shown as no description is available for a timetable set.
+
                 if (SelectedTimetable != null)
-                {
-                    ShowDetail(catalog.GetStringFmt("Timetable: {0}", SelectedTimetable), new string[0]);
-                }
+                    ShowDetail(catalog.GetStringFmt($"Timetable: {SelectedTimetable}"), SelectedTimetable.Briefing.Split('\n'));
+
                 if (SelectedTimetableTrain != null)
                 {
-                    ShowDetail(catalog.GetStringFmt("Train: {0}", SelectedTimetableTrain), SelectedTimetableTrain.ToInfo());
+                    var selectedTimetableTrainDescription = new string[2];
+                    selectedTimetableTrainDescription[0] = SelectedTimetableTrain.Briefing;
+
+                    ShowDetail(catalog.GetStringFmt($"Train: {SelectedTimetableTrain}"), selectedTimetableTrainDescription);
```

Comment:
```suggestion
                    ShowDetail(catalog.GetStringFmt("Train: {0}", SelectedTimetableTrain), SelectedTimetableTrain.ToInfo());
```
You've broken not only translations, but also timetable descriptions for trains without briefings. I suggest moving this logic to the `TimetableFileLite.TrainInformation.ToInfo()` method and updating it to support both kinds of trains.

Reply:
Patch (I apologize, I am not able to create a GitHub suggestion for it):
```diff
diff --git a/Source/Orts.Formats.OR/TimetableFile.cs b/Source/Orts.Formats.OR/TimetableFile.cs
index 5ee185143..4f19d7348 100644
--- a/Source/Orts.Formats.OR/TimetableFile.cs
+++ b/Source/Orts.Formats.OR/TimetableFile.cs
@@ -289,11 +289,10 @@ override public string ToString()

             public string[] ToInfo()
             {
-                string[] infoString = new string[] {
-                    Catalog.GetStringFmt("Start time: {0}", StartTime),
-                };
-
-                return (infoString);
+                if (string.IsNullOrEmpty(Briefing))
+                    return new[] { Catalog.GetStringFmt("Start time: {0}", StartTime) };
+                else
+                    return new[] { Catalog.GetStringFmt("Start time: {0}", StartTime), "", Briefing };
             }
         }
     }

```

Reply:
Will do.